spec/unit/actions/v3/service_instance_update_managed_spec.rb

```diff
@@ -542,225 +529,28 @@ module V3
                 expect_any_instance_of(ManagedServiceInstance).to receive(:update).
                   and_raise(Sequel::ValidationFailed.new(errors))
 
-                expect { action.try_update_sync }.
-                  to raise_error(V3::ServiceInstanceUpdateManaged::InvalidServiceInstance, 'blork is busted')
+                expect {
```

Comment:
old formatting looked better

Reply:
I've changed some other expectations to be more consistent:
- if everything fits into a single line: `expect { ... }.to ...`
- otherwise:
  ```
    expect {
      ...
    }.to ...
  ```